plugins/orchestrator/src/components/ExecuteWorkflowPage/ExecuteWorkflowPage.tsx

```diff
@@ -27,154 +30,187 @@ import {
   executeWorkflowWithBusinessKeyRouteRef,
   workflowInstanceRouteRef,
 } from '../../routes';
-import { BaseOrchestratorPage } from '../BaseOrchestratorPage/BaseOrchestratorPage';
-import { OrchestratorSupportButton } from '../OrchestratorSupportButton/OrchestratorSupportButton';
-import { WorkflowDialog } from '../WorkflowDialog';
-import { EditorViewKind } from '../WorkflowEditor';
-import { TitleFieldTemplate } from './TitleFieldTemplate';
-
-const WrappedForm = withTheme(require('@rjsf/material-ui-v5').Theme);
+import { getErrorObject } from '../../utils/error';
+import { BaseOrchestratorPage } from '../next/BaseOrchestratorPage';
+import SubmitBtn from '../SubmitBtn/SubmitBtn';
+import StepperForm from './StepperForm';
 
-interface ExecuteWorkflowPageProps {
+export interface ExecuteWorkflowPageProps {
   initialState?: Record<string, JsonValue>;
 }
 
-export const ExecuteWorkflowPage = (props: ExecuteWorkflowPageProps) => {
+const JsonTextAreaForm = ({
+  isExecuting,
+  handleExecute,
+}: {
+  isExecuting: boolean;
+  handleExecute: (
+    getParameters: () => Record<string, JsonValue>,
+  ) => Promise<void>;
+}) => {
+  const [jsonText, setJsonText] = React.useState('');
+  const theme = useTheme();
+  const getParameters = (): Record<string, JsonValue> => {
+    if (!jsonText) {
+      return {};
+    }
+    const parameters = JSON.parse(jsonText);
+    return parameters as Record<string, JsonValue>;
+  };
+
+  return (
+    <Grid container spacing={2}>
+      <Grid item xs={12}>
+        <Alert severity="info" style={{ width: '100%' }}>
+          <AlertTitle>
+            Couldn't find a valid JSON schema to display the input form.
+          </AlertTitle>
+          If you want to use a form to start the workflow, please provide a
+          valid JSON schema in the dataInputSchema property of your workflow
+          definition file. Alternatively, you can type below the input data in
+          JSON format.
+        </Alert>
+      </Grid>
+      <Grid item xs={12}>
+        <Box style={{ border: `1px solid ${theme.palette.border}` }}>
+          <Editor
+            language="json"
+            onChange={(value: string | undefined) => setJsonText(value || '')}
+            height="30rem"
+            options={{
+              lineNumbers: 'off',
```

Comment:
Wondering if it'd look better to enable line numbers.

Reply:
:+1: 
From the screenshot in the PR description, the first time I saw it I didn't get that the massive blank box 
below the "no input schema" warning was meant for entering a free-form JSON. (NB: Remember, users don't read long messages)

Reply:
WDYT?

```ts
options={{
  lineNumbers: 'on',
  glyphMargin: false,
  folding: true,
  minimap: { enabled: false },
}}
```

<img width="433" alt="image" src="https://github.com/caponetto/backstage-plugins/assets/638737/0151000a-c55e-44f7-b16e-c815a135ff6c">

Reply:
Looks good
this leaves only needing to configure 
minimap: {enabled: false}
the rest are there by default